src/main/java/io/openliberty/tools/intellij/actions/LibertyGeneralAction.java

```diff
@@ -27,15 +39,15 @@ public class LibertyGeneralAction extends AnAction {
     public void actionPerformed(@NotNull AnActionEvent e) {
         project = LibertyProjectUtil.getProject(e.getDataContext());
         if (project == null) {
-            String msg = "Unable to " + actionCmd + ": Could not resolve project. Ensure you run the Liberty action from the Liberty tool window";
+            String msg = LocalizedResourceUtil.getMessage("liberty.project.does.not.resolve", actionCmd);
             notifyError(msg);
             log.debug(msg);
             return;
         }
 
         buildFile = (VirtualFile) e.getDataContext().getData(Constants.LIBERTY_BUILD_FILE);
         if (buildFile == null) {
-            String msg = "Unable to " + actionCmd + ": Could not resolve configuration file for  " + project.getName() + ". Ensure you run the Liberty action from the Liberty tool window";
+            String msg = LocalizedResourceUtil.getMessage("liberty.build.file.does.not.resolve", actionCmd,project.getName());
```

Comment:
At some point someone is going to want to put a space after the comma "Cmd,proj"